<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModesWindow</class>
 <widget class="QDialog" name="ModesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Georgia</family>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="MinMaxButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">&lt;p&gt;PERSON&lt;b&gt; VS&lt;/b&gt; PERSON&lt;/p&gt;</string>
   </property>
   <property name="text">
    <string>
PERSON
VS
PERSON
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BetaAlphaButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">&lt;p&gt;PERSON
&lt;strong&gt;VS&lt;/strong&gt; 
COMPUTER&lt;/p&gt;</string>
   </property>
   <property name="text">
    <string>
PERSON
 VS
COMPUTER
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BettaAlphaIterativeButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>330</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">&lt;p&gt;COMPUTER
&lt;strong&gt;VS&lt;/strong&gt; 
COMPUTER&lt;/p&gt;</string>
   </property>
   <property name="text">
    <string>
COMPUTER
VS
COMPUTER
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
